From d2ed273c5c907c9fc627cf653ae6594feb7fb52a Mon Sep 17 00:00:00 2001
From: rmuxnet <armandas.kvietkus@proton.me>
Date: Fri, 3 Jul 2026 11:57:17 +0000
Subject: [PATCH] drm/amdgpu: log AUX transfer addr/ret/ack/buf in native read
 path
MIME-Version: 1.0
Content-Type: multipart/mixed; boundary="------------false"

This is a multi-part message in MIME format.
--------------false
Content-Type: text/plain; charset=UTF-8; format=fixed
Content-Transfer-Encoding: 8bit

---
 drivers/gpu/drm/amd/amdgpu/atombios_dp.c | 3 +++
 1 file changed, 3 insertions(+)


--------------false
Content-Type: text/x-patch; name="0132-drm-amdgpu-log-AUX-transfer-addr-ret-ack-buf-in-nati.patch"
Content-Transfer-Encoding: 8bit
Content-Disposition: attachment; filename="0132-drm-amdgpu-log-AUX-transfer-addr-ret-ack-buf-in-nati.patch"

diff --git a/drivers/gpu/drm/amd/amdgpu/atombios_dp.c b/drivers/gpu/drm/amd/amdgpu/atombios_dp.c
index 685309ca50cf15dba3290c44979691e29aa5634d..4501b636abfcedace13d6769babb04bc349e336c 100644
--- a/drivers/gpu/drm/amd/amdgpu/atombios_dp.c
+++ b/drivers/gpu/drm/amd/amdgpu/atombios_dp.c
@@ -173,6 +173,9 @@ amdgpu_atombios_dp_aux_transfer(struct drm_dp_aux *aux, struct drm_dp_aux_msg *m
 			tx_buf[3] |= tx_size << 4;
 		ret = amdgpu_atombios_dp_process_aux_ch(chan,
 						 tx_buf, tx_size, msg->buffer, msg->size, delay, &ack);
+		DRM_DEBUG_KMS("aux xfer: addr=0x%05x req=0x%x size=%zu ret=%d ack=0x%02x buf=%*ph\n",
+			      msg->address, msg->request, msg->size, ret, ack,
+			      (int)msg->size, msg->buffer);
 		break;
 	default:
 		ret = -EINVAL;

--------------false--


